client/lib/signup/page-builder.js

```diff
@@ -24,3 +25,8 @@ export function shouldEnterPageBuilder() {
 export function isEligibleForPageBuilder( segment, flowName ) {
 	return 'en' === getLocaleSlug() && 1 === segment && 'onboarding-for-business' === flowName;
 }
+
+export function isBlockEditorSectionInTest( state ) {
+	const isGutenberg = getSectionGroup( state ) === 'gutenberg';
+	return isGutenberg && isInPageBuilderTest();
```

Comment:
Why not just turn it into a one-liner whizzbang return doohickey?

`return 'gutenberg' === getSectionGroup( state ) && isInPageBuilderTest();`

Reply:
No real reason other than the first version of this had more conditions that got pared down to this.